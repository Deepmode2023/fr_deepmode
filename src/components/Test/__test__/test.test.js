import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import { Test } from "../Test";

describe("Page", () => {
  it("renders a heading", () => {
    const component = renderer.create(<Test />);

    expect(component).toMatchSnapshot();
  });
});
