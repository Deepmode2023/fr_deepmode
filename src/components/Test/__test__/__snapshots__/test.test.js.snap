// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders a heading 1`] = `
<div
  style={
    {
      "width": 300,
    }
  }
>
  <div
    className="flex flex-col gap-1 w-full"
  >
    <div
      className="flex border-b-2 gap-1 items-center"
      style={
        {
          "borderColor": "#6B7280",
        }
      }
    >
      <input
        className="w-full bg-transparent outline-none text-gray-700 "
        data-testid="input-basic-input"
        onBlur={[Function]}
        onChange={[Function]}
        type="email"
        value=""
      />
    </div>
    <div
      className="text-xs select-none overflow-hidden"
      style={
        {
          "color": "#6B7280",
        }
      }
    >
      <div
        data-testid="input-basic-placeholder"
        style={
          {
            "height": "0px",
          }
        }
      >
        Some place
      </div>
    </div>
  </div>
</div>
`;
